cclib/parser/psi4parser.py

```diff
@@ -824,11 +820,9 @@ def get_normalization_factor(exp, lx, ly, lz):
         #     2     0.00000000000000    -0.00979709321487     0.01460651641258
         #     3     0.00000000000000     0.00979709321487     0.01460651641258
 
-        if line.strip() == '## F-D gradient (Symmetry 0) ##':
-            next(inputfile)
-            self.skip_lines(inputfile, ['b'])
-            next(inputfile)
-            self.skip_lines(inputfile, ['b'])
+        if line.strip() in self.gradient_headers:

```

Comment:
I think reducing line count is great, but using the vars instead of strings here makes the parser less readable. Do you think it makes sense to put the parsing code in a helper method and call it differently in the two cases?